Bottom: c82596dbbebc8cf6633f7d5c88016cff86417b3d
Top:    3b558615d61c0132e4458b85bf113d9c07ee2c89
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 13:48:24 -0500

Refresh of wiki-update-entrypoint

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 81d9b004..25c1b3b1 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -58,7 +58,7 @@
     - bacula_restores
   - name: Configure Docker networking
     role: docker-networking
-    net_connection: '{{ daedalus_net_connection }}'
+    network_connection: '{{ daedalus_net_connection }}'
     subnet: '{{ daedalus_subnet }}'
     services:
     - svn
diff --git a/docker/docker-mediawiki/docker-entrypoint.sh b/docker/docker-mediawiki/docker-entrypoint.sh
index 0e8fc7e2..ff93cb5c 100755
--- a/docker/docker-mediawiki/docker-entrypoint.sh
+++ b/docker/docker-mediawiki/docker-entrypoint.sh
@@ -108,10 +108,8 @@ case ${1} in
     wiki)
         # verify permissions
         chown -R www-data:www-data ${WIKI_BASE_DIR}
-        # Apache gets grumpy about PID files pre-existing
-        rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     LocalSettings)
